### PR TITLE
Payment Emails: Give them their login e-mail somewhere

### PR DESCRIPTION
**Pull request by weedySeaDragon**
### PT Story: Use standard text to explain how to log in _and_ provide the user's login email as a reminder how to log in.
https://www.pivotaltracker.com/story/show/165126396

### Changes proposed in this pull request:
1.  Created a standard partials for common text:
    - logging in: the login page and user's login email
    - how to pay membership fee
    - how to pay H-markt fee
    - how to upload files
2. Revised emails to use the partials.  Emails had diverged, so I took this opportunity to revise them so they are standardized (using the partials).
3. removed no longer needed partials and locale entries
4. Locale entries

### Screenshots (Optional):

**Here is the current list of emails we send out via the alerts.** (This is the mailer preview page.) 
This PR only deals with those from the **Member mailer** and the **SHF Application Mailer.**
There are 10 emails to check, each in both Swedish (sv) and English (en).

list-of-emails--previewer

---

#### 1. Member mailer: app_no_uploaded_files

email-app-no-uploaded-files--sv-2

---

email-app-no-uploaded-files--en

---
---

#### 2. Member mailer: company_info_incomplete

email-co-info-incomplete--sv-2

---

email-co-info-incomplete--en

---
---

#### 3. Member mailer: h_branding_fee_past_due

email-hbranding-fee-past-due--sv-2

---

email-hbranding-fee-past-due--en

---
---

#### 4. Member mailer: h_branding_fee_will_expire

email-hbranding-will-expire--sv-2

---

email-hbranding-will-expire--en

---
---

#### 5. Member mailer: membership_expiration_reminder

email-membership-expiration-reminder--sv-2

---

email-membership-expiration-reminder--en

---
---

#### 6. Member mailer: membership_granted

email-membership-granted--sv-2

---

email-membership-granted--en

---
---

#### 7. Member mailer: membership_lapsed

email-membership-lapsed--sv-2

---

email-membership-lapsed--en

---
---

#### 8. SHF Application mailer: acknowledge_received_no_files_uploaded

email-ack-received-no-files-uploaded--sv-2

---

email-ack-received-no-files-uploaded--en

---
---

#### 9. SHF Application mailer: acknowledge_received_with_files_uploaded

email-ack-received-files-uploaded--sv-2

---

email-ack-received-files-uploaded--en

---
---

#### 10. SHF Application mailer: app_approved

email-app-approved--sv-2

---

email-app-approved--en

---
---

### Ready for review:
@thesuss

### CODE REVIEW

File: config/locales/sv.yml [hunk 1]
```diff
@@ -1058,8 +1058,18 @@ sv:
 
 
   mailers:
-    login_here: "Klicka här för att gå till inloggningssidan:"
-    may_need_to_login_first: "Du kan behöva logga in först.  Här är inloggningssidan:"
+    login_here: "Klicka här för att gå till inloggningssidan."
+    your_login_email_is: "Your login email is "
```
Comment (thesuss) [inline config/locales/sv.yml]: Din e-post för att logga in är:

File: config/locales/sv.yml [hunk 2]
```diff
@@ -1058,8 +1058,18 @@ sv:
...
+
+    how_to_pay_membership: "How to Pay Your Membership Fee:"
```
Comment (thesuss) [inline config/locales/sv.yml]: Så här betalar du medlemsavgiften:

File: config/locales/sv.yml [hunk 3]
```diff
@@ -1058,8 +1058,18 @@ sv:
...
+    pay_membership_step: "Once you are logged in, go to your account page to pay."
```
Comment (thesuss) [inline config/locales/sv.yml]: När du är inloggad hittar du betalknappen på ditt konto.

File: config/locales/sv.yml [hunk 4]
```diff
@@ -1058,8 +1058,18 @@ sv:
...
+    here_is_acct_page: "Here is your account page: "
```
Comment (thesuss) [inline config/locales/sv.yml]: Här är en länk till ditt konto:

File: config/locales/sv.yml [hunk 5]
```diff
@@ -1058,8 +1058,18 @@ sv:
...
+
+    how_to_pay_hmarkt: "How to Pay the H-Markt License Fee:"
```
Comment (thesuss) [inline config/locales/sv.yml]: Så här betalar du H-märkespremien:

File: config/locales/sv.yml [hunk 6]
```diff
@@ -1058,8 +1058,18 @@ sv:
...
+    pay_hmarkt_step: "När du är inloggad ser du när din H-märkningspremie går ut och hur du betalar på din företagssida."
+    here_is_co_page: "Här är länken där du kan redigera ditt företag: "
+    h_brand_exp_shown: "THE H-MARKT LICENSE EXPIRATION WILL BE ON THE COMPANY PAGE."
```
Comment (thesuss) [inline config/locales/sv.yml]: **_I think this is said in "    pay_hmarkt_step:"_**

**Review by thesuss — CHANGES_REQUESTED** [review]

**Comment by thesuss** [comment]
There are a bunch of language things, but I can do them separate

**Review by thesuss — APPROVED** [review]

**Review by patmbolger — APPROVED** [review]

**Comment by weedySeaDragon** [comment]
Interesting that the cucumber tests failed on this one: 
```
  @selenium @time_adjust
  Scenario: Search by category and region                                                   # features/search_companies.feature:170
    Given I am Logged out                                                                   # features/step_definitions/user_steps.rb:27
    And I am on the "landing" page                                                          # features/step_definitions/navigation_steps.rb:1
    Then I select "Groomer" in select list t("activerecord.models.business_category.one")   # features/step_definitions/basic_steps.rb:76
    Then I select "Västerbotten" in select list t("activerecord.attributes.company.region") # features/step_definitions/basic_steps.rb:76
    And I click on t("search")                                                              # features/step_definitions/basic_steps.rb:7
    And I should not see "HappyMutts" in the list of companies                              # features/step_definitions/company_steps.rb:107
    And I should not see "We Luv Dogs" in the list of companies                             # features/step_definitions/company_steps.rb:107
    And I should not see "Barky Boys" in the list of companies                              # features/step_definitions/company_steps.rb:107
    And I should not see "Dogs R Us" in the list of companies                               # features/step_definitions/company_steps.rb:107
    Then I select "Stockholm" in select list t("activerecord.attributes.company.region")    # features/step_definitions/basic_steps.rb:76
    And I click on t("search")                                                              # features/step_definitions/basic_steps.rb:7
    And I should see "Barky Boys"                                                           # features/step_definitions/assertion_steps.rb:81
    Then I select "Sweden" in select list t("activerecord.attributes.company.region")       # features/step_definitions/basic_steps.rb:76
    And I click on t("search")                                                              # features/step_definitions/basic_steps.rb:7
    And I should see "We Luv Dogs"                                                          # features/step_definitions/assertion_steps.rb:81
      expected all ajax requests to be completed after scenario, but some ajax requests were still running. (RuntimeError)
      ./features/support/hooks.rb:29:in `After'
```

First time I've seen that scenario fail based on ajax.  The scenario that does the nearly same thing (in the same feature file) passed.

So am going to merge this.

**Pull request merged**